src/Hl7.Fhir.Specification/Validation/XmlValidationExtensions.cs

```diff
@@ -6,7 +6,9 @@
  * available at https://raw.githubusercontent.com/FirelyTeam/firely-net-sdk/master/LICENSE
  */
 
-#nullable enable
+#nullable enable annotations
+#nullable disable warnings
```

Comment:
At the bottom of this file, there is a `#nullable restore` directive. Should this be changed as well?

Reply:
No, because:

> #nullable restore: Restores the nullable annotation context and nullable warning context to the project settings.